Tasks/ANTPreview/task.json

```diff
@@ -11,7 +11,7 @@
                   ],    
     "author": "Microsoft Corporation",
     "version": {
-        "Major": 0,
+        "Major": 1,
```

Comment:
check description ... the version should start from 1. We were starting from 0, so i fixed that.